src/logion/model/verifiedthirdpartynomination.model.ts

```diff
@@ -0,0 +1,85 @@
+import { injectable } from 'inversify';
+import { Entity, PrimaryColumn, Repository } from "typeorm";
+import { appDataSource, requireDefined } from "@logion/rest-api-core";
+import { LocRequestAggregateRoot, LocRequestRepository } from './locrequest.model';
+
+export interface VerifiedThirdPartyNominationId {
```

Comment:
IMHO each `VerifiedThirdPartyNomination*` should be named `VerifiedThirdPartySelection*`.

Reply:
You're right. Just pushed a fix.